src/test/java/api/items/ItemApiExamples.java

```diff
@@ -1383,6 +1429,145 @@ public void cannotRemoveHRID() throws Exception {
+      "Unrecognized name"
+    ));
+  }
+
+  @Test
+  public void cannotUpdateItemWithUnrecognisedStatusName() throws Exception {
+    JsonObject createdInstance = createInstance(smallAngryPlanet(UUID.randomUUID()));
+
+    UUID holdingId = holdingsStorageClient.create(
+      new HoldingRequestBuilder()
+        .forInstance(UUID.fromString(createdInstance.getString("id"))))
+      .getId();
+
+    IndividualResource postResponse = itemsClient.create(new ItemRequestBuilder()
+      .forHolding(holdingId)
+      .temporarilyInReadingRoom());
+
+    JsonObject createdItem = postResponse.getJson();
+    assertThat(createdItem.getJsonObject("status").getString("name"),
+      is(ItemStatusName.AVAILABLE.value()));
+
+    JsonObject updatedItem = createdItem.copy()
+      .put("status", new JsonObject().put("name", "Unrecognized name"));
+
+    Response updateResponse = updateItem(updatedItem);
+    assertThat(updateResponse, hasValidationError(
+      "Undefined status specified",
+      "status.name",
+      "Unrecognized name"
+    ));
+  }
+
+  @Test
+  public void cannotRemoveStatusFromItem() throws Exception {
+    JsonObject createdInstance = createInstance(smallAngryPlanet(UUID.randomUUID()));
+
+    UUID holdingId = holdingsStorageClient.create(
+      new HoldingRequestBuilder()
+        .forInstance(UUID.fromString(createdInstance.getString("id"))))
+      .getId();
+
+    IndividualResource postResponse = itemsClient.create(new ItemRequestBuilder()
+      .forHolding(holdingId)
+      .temporarilyInReadingRoom());
+
+    JsonObject createdItem = postResponse.getJson();
+    assertThat(createdItem.getJsonObject("status").getString("name"),
+      is(ItemStatusName.AVAILABLE.value()));
+
+    JsonObject updatedItem = createdItem.copy();
+    updatedItem.remove("status");
+
+    Response updateResponse = updateItem(updatedItem);
+    assertThat(updateResponse,
+      hasValidationError("Status is a required field", "status", null)
+    );
+  }
+
+  @Test
+  public void cannotRemoveStatusNameFromItem() throws Exception {
+    JsonObject createdInstance = createInstance(smallAngryPlanet(UUID.randomUUID()));
+
+    UUID holdingId = holdingsStorageClient.create(
+      new HoldingRequestBuilder()
+        .forInstance(UUID.fromString(createdInstance.getString("id"))))
+      .getId();
+
+    IndividualResource postResponse = itemsClient.create(new ItemRequestBuilder()
+      .forHolding(holdingId)
+      .temporarilyInReadingRoom());
+
+    JsonObject createdItem = postResponse.getJson();
+    assertThat(createdItem.getJsonObject("status").getString("name"),
+      is(ItemStatusName.AVAILABLE.value()));
+
+    JsonObject updatedItem = createdItem.copy()
+      .put("status", new JsonObject());
+
+    Response updateResponse = updateItem(updatedItem);
+    assertThat(updateResponse,
+      hasValidationError("Status is a required field", "status", null)
+    );
+  }
+
+  @Test
+  public void canHandleStatusDateProperty() throws Exception {
```

Comment:
Please do not use `throws Exception` it masked all of the relevant checked exceptions and makes refactoring the tests harder

Reply:
Yes, they mask, but there is another side, when we specify all the "standard" test checked exceptions, which in a fact do not give me any useful info (since we don't execute test methods in code), but instead have to maintain (e.g. test were refactored to use client instead of okapiClient, and now `MalformedURLException` have to be removed) and makes tests cluttered a bit.

But, it is just my opinion.

Reply:
I think your point about useful information is interesting. It is annoying that these exceptions propagate. In general I think this is a flaw of checked exceptions in general.

I'm going through a refactoring of mod-circulation's tests to use Rest Assured, which affects a lot of the `standard` exceptions thrown by the tests. The use of `Exception` has made that harder.

Personally, I like that we need to remove the exceptions when they are no longer thrown, it is a clear indicator of what exceptions are thrown.

Do you use exception because it means that test maintainers don't have to think about checked exceptions?

Reply:
@bohdan-suprun You are correct that this is a more involved topic than for this pull request, please don't make any changes to the checked exceptions

Reply:
Yes, I prefer single `Exception` here because we never call the test method within another method, so we don't need to definitely know what may be thrown, and JUnit takes care about handling it.

But if it makes harder to introduce the new approach, then I will expand the throws clause.

Reply:
@bohdan-suprun It does make refactoring harder because it impedes some of the automated tools from detecting the correct exceptions.

This can be addressed during a future change if necessary.